README.md

```diff
@@ -0,0 +1,55 @@
+# Checkmark Backend
+This beginner-friendly project is primarly the backend part of the Checkmark (To-Do list) project. It gives users the feature to add and manage various daily tasks. Moreover, it is highly suitable for beginners to jumpstart their opensource journey. 
+
+
+## Setup
+Follow these steps to set up and run the Checkmark project locally:
+
+### Prerequisites
+Before you begin, ensure you have the following installed on your machine:
+
++ **Node.js (version 14.x or later)**
++ **npm (Node Package Manager)**
+
+### Quick Setup
+
+Run the following commands in your terminal to set up and run the Checkmark project:
+
+```bash
+
+# Clone the repository 
+git clone https://github.com/your-username/checkmark-backend.git
```

Comment:
Yes, that's correct and that is already present in the `CONTRIBUTING.md`. But it'll be better to just add the one step here as well since it is easily visible to beginners.

Reply:
Alright, I will get it done !